monai/transforms/transforms.py

```diff
@@ -72,12 +72,12 @@ class Flip:
     """
 
     def __init__(self, axes=None):
-        assert isinstance(axes, int) or isinstance(axes, tuple) and all(isinstance(n, int) for n in axes), \
+        assert isinstance(axes, (int, list, tuple)) and all(isinstance(n, int) for n in axes), \
```

Comment:
I think this line is still not correct:
```py
all(isinstance(n, int) for n in axes)
```
If axes is an int number, it's **not iterable**, your unit test doesn't cover int number case.

Reply:
Fixed.